### PR TITLE
PMM-11207 migrate pmm-update to the monorepo

### PR DESCRIPTION
**Pull request by ademidoff**
https://jira.percona.com/browse/PMM-11207

- https://github.com/percona/pmm/pull/1485

### CODE REVIEW

File: .github/workflows/jenkins-dispatch.yml [hunk 1]
```diff
@@ -1,37 +1,30 @@
...
-        uses: percona-platform/checkout@v3
+        uses: actions/checkout@v3
         with:
-          submodules: 'recursive'
-
-      - name: install npm
-        uses: percona-platform/setup-node@v3
-        with:
-          node-version: '14'
+          submodules: "recursive"
```
Comment (ademidoff) [inline .github/workflows/jenkins-dispatch.yml:27]: Note: NodeJS is provisioned on all runners.

File: .github/workflows/jenkins-dispatch.yml [hunk 2]
```diff
@@ -1,37 +1,30 @@
-on: 
+on:
```
Comment (ademidoff) [inline .github/workflows/jenkins-dispatch.yml:1]: Sorry, a bit of auto-formatting :)

**Review by ghost — APPROVED** [review]

**Comment by ademidoff** [comment]
@michal-kralik I guess we should make it a feature build. WDYT?

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2941-bd0541c
client docker - perconalab/pmm-client-fb:PR-2941-bd0541c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2941-bd0541c.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2941-bd0541c&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2941-bd0541c.tar.gz

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/25989/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2941-78a0b05
client docker - perconalab/pmm-client-fb:PR-2941-78a0b05
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2941-78a0b05.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2941-78a0b05&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2941-78a0b05.tar.gz

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/25990/

**Pull request merged**